usr/share/drlm/lib/help-functions.sh

```diff
@@ -290,6 +290,7 @@ function runbackuphelp() {
   printf "\n"
   printf "$(tput bold)Options:\n$(tput sgr0)"
   printf '%-25s %-45s\n' "-C|--config" "Select Client backup configuration"
```

Comment:
We may find a better description? for now we can keep it and may or may not find a better one.